src/drawers/symbolDrawer.ts

```diff
@@ -3,10 +3,95 @@
  * @license MIT
  */
 
+import { Dataset } from "../core/dataset";
+import { AttributeToAppliedProjector, IAccessor } from "../core/interfaces";
+import { SymbolFactory } from "../core/symbolFactories";
+import { CanvasBuffer } from "./canvasBuffer";
+import { CanvasDrawStep, ContextStyleAttrs, resolveAttributesSubsetWithStyles, styleContext } from "./canvasDrawer";
 import { SVGDrawer } from "./svgDrawer";
 
 export class SymbolSVGDrawer extends SVGDrawer {
-  constructor() {
-    super("path", "symbol");
-  }
+    constructor() {
+        super("path", "symbol");
+    }
+}
+
+// for testing use of buffer
+export let buffer: CanvasBuffer;
```

Comment:
the buffer is only used for copying the symbol over and over. I didn't want to create buffers for every combination of symbol+color+size, so instead i just re-use this one buffer if all those are the same as previous